tools/doc/generate.js

```diff
@@ -82,7 +82,7 @@ async function main() {
   const content = await unified()
     .use(replaceLinks, { filename, linksMapper })
     .use(markdown)
-    .use(html.preprocessText)
+    .use(html.preprocessText, { nodeVersion })
```

Comment:
Can you undo the mode change for this file?

Reply:
Yeah, did not mean to commit that. I think I just changed it back.